registry/Dockerfile

```
@@ -0,0 +1,30 @@
+# Registry container image
+
+# Stage1: build from source
+FROM quay.io/cybozu/golang:1.15-bionic AS build
```

```suggestion
FROM quay.io/cybozu/golang:1.15-focal AS build
```

To use the same base image as that of stage 2.